src/block.js

```diff
@@ -115,4 +116,25 @@ Block.prototype.toHex = function (headersOnly) {
   return this.toBuffer(headersOnly).toString('hex')
 }
 
+Block.calculateTarget = function (bits) {
+  var target = new Buffer(32)
+  target.fill(0)
+
+  var exponent = ((bits & 0xff000000) >> 24) - 3
+  var value = bits & 0x00ffffff
```

Comment:
Its unclear if this should be `bits & 0x007fffff` or `0x00ffffff`.
I'm happy with this as is for now, will wait to enforce that it verifies for the mainnet blockchain before merge.